### ISSUE TITLE
[Feature request] Async PR comments

### ISSUE THREAD

**Issue by rlogiacco** [go-gitea/gitea#12889]
I wish to ask if we can have a different UX with regards to commenting PRs, where we do not reload the page at every comment, but instead send the comments async... 

On big PRs it takes minutes to reload the page after each and every comment and even on small PR's the screen flickers back & forth with the reload having to move back to the correct position once page load completes.

We can then use message boxes to report the "comment save confirmed!" or "comment save failed!" to avoid having users write tons of comments blindly just to discover the backend was not reachable for some reason...

**Comment by rlogiacco** [comment]
Is it anything a non Go developer can do to help? :-D

**Comment by zeripath** [comment]
This is entirely something a non-go developer can do.

Sending a POST to /api/v1/repos/{owner}/{repo}/issues/{index}/comments 
 (https://try.gitea.io/api/swagger#/issue/issueCreateComment) allows you to POST a new comment.

You can get a render of the comment from POST ing to /api/markdown as per the preview.

It would be better if we had media types support a la https://docs.github.com/en/rest/overview/media-types but you can get away with using the POST to markdown as above to get a render for at least an initial implementation.

**Comment by wxiaoguang** [comment]
When reviewing big PRs, you could "Start review" and IIRC there is no page reload?

**Comment by delvh** [comment]
Yes, IIRC in the old times of like 4 years ago, that did actually happen.
But nowadays it's probably fixed already

**Comment by wxiaoguang** [comment]
Let's close it as "completed", feel free to reopen if there is still any problem.
